media/starboard/starboard_renderer.cc

```diff
@@ -183,16 +183,29 @@ void StarboardRenderer::SetCdm(CdmContext* cdm_context,
     return;
   }
 
-  drm_system_ = cdm_context->GetSbDrmSystem();
+  cdm_context_ = cdm_context;
+  drm_system_ = cdm_context_->GetSbDrmSystem();
   std::move(cdm_attached_cb).Run(true);
   LOG(INFO) << "CDM set successfully.";
+
+  if (state_ != STATE_INIT_PENDING_CDM) {
```

Comment:
Could `state_` be `STATE_UNINITIALIZED` here?

Reply:
If it is `STATE_UNINITIALIZED`, then we let `Initialize()` to create player bridge. On Chromium, `SetCdm` is called before `Initialize`.

https://source.chromium.org/chromium/chromium/src/+/main:media/base/pipeline_impl.cc;l=1186;drc=5d0194cbc9cddc2ad096da32f16d7d1eac5d4a34;bpv=0;bpt=1